api/v1/report.proto

```diff
@@ -114,33 +114,32 @@ message LogEntry {
     EMERGENCY = 800;
   }
 
+  // The name of the LogEntryDescriptor providing the schema for this log entry.
+  string descriptor_name = 1;
```

Comment:
Depends on the case.

The metrics, quotas, and logs that the proxy will use will require well-known descriptor names. Those probably want to be qualified with something like proxy/<name>. For those descriptors for things that only the service itself will reference, such as for resource quotas or principals for permission checks, then those names are entirely under operator control and so don't need to be qualified.